pkg/services/sqlstore/migrations/annotation_mig.go

```diff
@@ -109,4 +110,51 @@ func addAnnotationMig(mg *Migrator) {
 	//
 	updateEpochSql := "UPDATE annotation SET epoch = (epoch*1000) where epoch < 9999999999"
 	mg.AddMigration("Convert existing annotations from seconds to milliseconds", NewRawSqlMigration(updateEpochSql))
+
+	//
+	// 6.3: Make Regions a single annotation row
+	//
+	mg.AddMigration("Add epoch_end column", NewAddColumnMigration(table, &Column{
+		Name: "epoch_end", Type: DB_BigInt, Nullable: false, Default: "0",
+	}))
+	mg.AddMigration("Add index for epoch_end", NewAddIndexMigration(table, &Index{
+		Cols: []string{"epoch_end"}, Type: IndexType,
```

Comment:
Think this index isn't useful since that would only be in use when filter annotations only by `epoch_end`. What we probably need is an index combined of org_id, epoch and epoch_end. 

Adding an index is maybe something that we can wait with to add when we see that it's actually needed?

Reply:
I just changed it to:
```
	mg.AddMigration("Add index for epoch_end", NewAddIndexMigration(table, &Index{
		Cols: []string{"org_id", "epoch", "epoch_end"}, Type: IndexType,
	}))
```
But we should for sure check if this actually gets used in the real query!